index.test.js

```diff
@@ -247,7 +247,151 @@ describe('Deploy to ECS', () => {
             deploymentId: 'deployment-1',
             $waiter: {
                 delay: 15,
-                maxAttempts: 400
+                maxAttempts: (30 + 90) * 4
```

Comment:
nit: is it possible to parameterize these values to make it explicit what they're from? (example: i don't know what significance '90' has here. The DG wait time?)

Reply:
will do :)